internal/pe/README

```diff
@@ -0,0 +1,7 @@
+This is a copy of debug/pe from go 1.14.2 (96745b980cfde139e8611772e2bc0c59a8e6cdf7)
```

Comment:
when we do this kind of thing (we had to in the past for some things in snapd itself) we tend to call the directory something like pe14